infrastructure/local/docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
 services:
   auth:
-    image: ghcr.io/interledger/rafiki-auth:latest
```

Comment:
I vote leaving the `image` to give users the option of pulling or building the images (`pnpm localenv up` / `pnpm localenv up --build`)

Reply:
Afaik, the `--build` flag only builds those images that actually have the `- build` property in the docker-compose file. Without the `--build`, it first checks if there are already existing images. And with just `-image` property in the docker-compose, it will never build.

Reply:
Correct.
I think the docker compose files should have both `build` and `images`.